src/applications/model/crazytaxi-streaming-server.cc

```diff
@@ -24,6 +24,7 @@
 #include "seq-ts-header.h"
```

Comment:
We don't need to include the `seqTsHeader` header anymore, right?

Reply:
yeah
right